modules/nf-core/biohansel/main.nf

```diff
@@ -0,0 +1,45 @@
+process BIOHANSEL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::bio_hansel=2.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bio_hansel:2.6.1--py_0':
+        'quay.io/biocontainers/bio_hansel:2.6.1--py_0' }"
+
+    input:
+    tuple val(meta), path(seqs)
+    val scheme
+    path scheme_metadata
+
+    output:
+    tuple val(meta), path("${prefix}-summary.txt")       , emit: summary
+    tuple val(meta), path("${prefix}-kmer-results.txt")  , emit: kmer_results
+    tuple val(meta), path("${prefix}-simple-summary.txt"), emit: simple_summary
+    path "versions.yml"                                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def scheme_metadata_opt = scheme_metadata ? "--scheme-metadata ${scheme_metadata[0]}" : ""
```

Comment:
What is the reason for taking the first value in the channel here?

Reply:
honestly, probably just old habit, will clean it up

Reply:
No problem, I was just curious what the reason was :stuck_out_tongue: